src/win/d3d_disp.cpp

```diff
@@ -39,6 +39,24 @@
 
 #include "d3d.h"
 
+#ifdef ALLEGRO_HAVE_VERSIONHELPERS_H
+#include <VersionHelpers.h>
+#else
+// Based on the code in <VersionHelpers.h> but that header is not 
+// available on Windows 7 SDK and earlier.
+static BOOL IsWindowsVistaOrGreater() {
```

Comment:
It feels like it'd be simpler to just always use this implementation and not worry about VersionHelpers.h. What do you think? If we do that, could you rename it to `is_windows_vista_or_greater` to match our style?

Reply:
That makes sense, I will do it. The other advantage is that the changes to CMakeLists.txt and alplatf.h.cmake are no longer needed.